pkg/costmodel/allocation.go

```diff
@@ -615,7 +620,8 @@ func (cm *CostModel) computeAllocation(start, end time.Time, resolution time.Dur
 	applyRAMBytesRequested(podMap, resRAMRequests, podUIDKeyMap)
 	applyRAMBytesUsedAvg(podMap, resRAMUsageAvg, podUIDKeyMap)
 	applyRAMBytesUsedMax(podMap, resRAMUsageMax, podUIDKeyMap)
-	applyGPUUsageAvg(podMap, resGPUsUsageAvg, podUIDKeyMap)
+	applyGPUUsage(podMap, resGPUsUsageAvg, podUIDKeyMap, GpuUsageConfig{Mode: "Average"})
```

Comment:
This is good, but let's make average and max a constant in the allocation_helpers.go. 

I dont feel there is a need for GPUUsageConfig struct with just mode string. maybe the constant can reside in the same file allocation.go WDYT?